server/api.proto

```diff
@@ -888,13 +888,58 @@ message TopicPresence {
 }
 
 /**
+ * PropertyMap is a core domain type respresenting a single user property
```

Comment:
The comment references `PropertyMap` but the message name is `PropertyPair`, I think there was a change and these are no longer in sync?